cmd/main.go

```diff
@@ -0,0 +1,116 @@
+// Package cmd implements the commands supported by opfcli.
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/operate-first/opfcli/utils"
+)
+
+const defaultAppName = "cluster-scope"
+const namespacePath = "base/core/namespaces"
+const groupPath = "base/user.openshift.io/groups"
+const componentPath = "components"
+const componentRelPath = "../../../../components"
+
+var config = viper.New()
+
+var cfgFile string
+var appName string
+var repoDirectory string
+
+var rootCmd = &cobra.Command{
+	Use:   "opfcli",
+	Short: "A command line tool for Operate First GitOps",
+	Long: `A command line tool for Operate First GitOps.
+
+Use opfcli to interact with an Operate First style Kubernetes
+configuration repository.`,
+}
+
+// Execute processes the opfcli command line.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		log.Fatalf("Error: %s", err)
+	}
+}
+
+func init() {
+	loglevel, err := strconv.Atoi(os.Getenv("OPF_LOGLEVEL"))
```

Comment:
this env var doesn't like it's mentioned in the readme or anywhere else

Reply:
I've added a note to the configuration.